src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryDate.cs

```diff
@@ -656,5 +656,17 @@ public static FormulaValue TimeZoneOffset(EvalVisitor runner, EvalVisitorContext
                     return CommonErrors.InvalidDateTimeError(irContext);
             }
         }
+
+        private static bool TryGetInt(double inputValue, out int outputValue)
+        {
+            outputValue = 0;
+            if (inputValue < -32767d || inputValue > 32767d)
```

Comment:
>32767d [](http://example.com/codeflow?start=53&length=6)

nit: 
This is 16-bit (2^16 -1). 
int are 32 bit. 

Should we use int.MaxValue?

Reply:
remove this helper and use the helper in librarytext.